pkg/multitenant/tenantcapabilities/tenantcapabilitieswatcher/testdata/basic

```diff
@@ -0,0 +1,126 @@
+# Test basic tenant capability watcher functionality.
+
+start
+----
+ok
+
+updates
+----
+
+update-state
```

Comment:
Why is there no "Complete Update" here? It's completely updated, right? To an empty table.